include/z64transition_instances.h

```diff
@@ -23,6 +23,13 @@ typedef struct {
     /* 0xDC */ u16* zBuffer;
 } TransitionTile; // size = 0xE0
 
+typedef enum {
+    /* 0 */ TRANS_TILE_OFF, // Inactive, do nothing
+    /* 1 */ TRANS_TILE_SETUP, // Save the necessary buffers
+    /* 2 */ TRANS_TILE_PROCESS, // Initialize the transition
+    /* 3 */ TRANS_TILE_READY // The transition is ready, so will update and draw each frame
+} TransitionTileState;
```

Comment:
I think this is more general than transition instances (part of scene transitions), there's code that works outside of scene transitions

I would probably put this in its own transitiontile header

(this is also not used and not functional so may as well separate it)